libraries/csharp-dotnet-core2-buildtime/context/build

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
fyi - we'll also want the ability to pass in an environment variable to set a mirror for nuget if that's a possbility (i don't know the build tool)

Reply:
oh yea not a bad idea, I'll make sure to add

Reply:
I'll add this in a second QoL PR, closing for now - but I'll comment.